lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,25 @@
+desc 'refresh will merge your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git pull origin <current_branch>\n
+  \t$ git push origin <current_branch>\n
+  \t$ git merge <base_branch>\n
+LONGTIME
+arg_name '[remote_location] - remote repository name to fetch updates from (origin by default), [base_branch] - branch that you want to merge with (master by default)'
+command :refresh do |c|
+  c.flag [:r,:remote], default_value: 'origin'
+  c.flag [:b,:branch], default_value: 'master'
+  c.action do |global_options, options, args|
+    # usage: git reflow refresh [remote_location - defaults to origin] [base_branch_name - defaults to master]
+    # defaults remote_location to origin
+    refresh_options = {
+      'remote' => global_options.key?[:remote] ? global_options[:remote] : 'origin',
```

Comment:
since `global_options` is a hash it will just return `nil` if there is no key, so this can change to: `'remote' => global_options[:remote] || 'origin',`